compiler-rt/lib/builtins/cpu_model.c

```diff
@@ -1133,6 +1133,9 @@ typedef struct __ifunc_arg_t {
 #ifndef HWCAP2_SME_F64F64
 #define HWCAP2_SME_F64F64 (1 << 25)
 #endif
+#ifndef HWCAP2_SME_FA64
+#define HWCAP2_SME_FA64 (1 << 26)
```

Comment:
Where did you get the `1 << 26` from? I think it should be `30`.

Reply:
This comes from here https://github.com/torvalds/linux/blob/master/arch/arm64/include/uapi/asm/hwcap.h so it is in fact 30.